dev-tools/prepare_release_create_release_version.py

```diff
@@ -142,4 +142,4 @@ def find_release_version():
   shortHash = subprocess.check_output('git log --pretty=format:"%h" -n 1', shell=True)
   print('')
   print('*** To create a release candidate run: ')
-  print('  mvn clean deploy -Prelease -DskipTests -Dgpg.keyname="$GPG_KEY_ID" -Dgpg.passphrase="$GPG_PASSPHRASE" -Dpackaging.rpm.rpmbuild=/usr/bin/rpmbuild -Delasticsearch.s3.repository=s3://download.elasticsearch.org/elasticsearch/staging/%s' % (shortHash.decode('utf-8')))
```

Comment:
sorry for that. saturday morning reviews are not my thing. Due to the potential problem of having to enter the passphrase more than once, I'd suggest we print out two different mvn lines and let the user choose...